react-client/components/requestUtils/frameBufferFactory.ts

```diff
@@ -0,0 +1,11 @@
+const trajectoryUrl = 'http://localhost:5000/trajectory';
```

Comment:
Apologies for this poorly named file - must have forgotten to change its name after some iterations of refactoring :(